server/services/lobby_service.go

```diff
@@ -97,6 +98,28 @@ func (l *LobbyServiceImpl) ReadyPlayer(lobbyID string, readyMsg pogo.PlayerReady
 	return *lobby, nil
 }
 
+// IsReady checks the ready status of all players, if any are false, returns false
+func (l *LobbyServiceImpl) IsReady(lobbyID string) (bool, bool) {
```

Comment:
This is the main addition.  A ready checker that makes sure there are enough players and that they are all readied up.